### PR TITLE
recordDetailsFSC : Few Enhancements

### PR DESCRIPTION
**Pull request by mhamzas**
- Save and Cancel Button Labels are now changeable
- Option to hide CANCEL button
- Column Size option (Show all fields in 1 -> 12 sections)
- Disable Toast Option
- Removed slds-box class to remove double boxes under flow screen
- label-stacked variant added on input fields.

### CODE REVIEW

File: flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.js-meta.xml [hunk 1]
```diff
@@ -22,7 +22,13 @@
...
+            <property name="showCancelButton" type="Boolean" label="Show Cancel Button?" description="This will hide cancel button if set FALSE"/>
             <property name="isCancelButton" type="Boolean" role="outputOnly" label="Cancel Button Selected" description="This value will be set to True is the user selected the Cancel button."/>
+            <property name="SaveBtnLabel" type="String" label="Save Button Label" description="Input - This is the next/submit button label"/>
+            <property name="CancelBtnLabel" type="String" label="Cancel Button Label" description="Input - This is the previous/cancel button label"/>
+            <property name="columnsize" type="Integer" label="No. of Columns" default="2" description="Input - This will represent number of columns for View/Edit mode"/>
+            <property name="showtoast" type="Boolean" label="Show Toast on Success?" description="If True, Toast will be trigger on success"/>
```
Comment (alexed1) [inline flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.js-meta.xml:30]: "will be triggered"

File: flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.js-meta.xml [hunk 2]
```diff
@@ -22,7 +22,13 @@
             <property name="flowNavigationOnSave" type="Boolean" role="inputOnly" label="Flow - Navigation on Save?" description="Check if you would like the flow to navigate to the Next screen (or Finish) when clicking Save.  For best results, hide the page footer in the flow screen."/>
             <property name="flowNavigationOnCancel" type="Boolean" role="inputOnly" label="Flow - Navigation on Cancel?" description="Check if you would like the flow to navigate to the Previous or Next screen when clicking Cancel.  For best results, hide the page footer in the flow screen."/>
             <property name="flowNavigationOnCancelDirection" type="String" role="inputOnly" label="Flow - Navigation Direction on Cancel (Previous or Next)" description="Specify Previous or Next for the flow navigation direction after the Cancel button is clicked."/>
+            <property name="showCancelButton" type="Boolean" label="Show Cancel Button?" description="This will hide cancel button if set FALSE"/>
```
Comment (alexed1) [inline flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.js-meta.xml:25]: Some properties are so simple and obvious a description doesn't provide any new information

File: flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html [hunk 3]
```diff
@@ -1,6 +1,9 @@
 <!-- 
 Lightning Web Component for Flow Screens:       recordDetailFlowScreenComponent
 
+4/29/20 -   Eric Smith -    Version 1.6
```
Comment (alexed1) [inline flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html:4]: You should add a section summarizing your changes. You deserve the credit!

File: flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html [hunk 4]
```diff
@@ -1,6 +1,9 @@
 <!--
```
Comment (alexed1) [inline flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html:1]: These are some nice enhancements, Hamzas, and I'm glad you're building on your initial contribution. What I'd like you to do is create a draft blog post that describes the new features, with images. If you give me an email address here, I'll invite you as an Author to post on unofficialsf.com. I'll edit the post so don't worry about it being perfect English.  If the feature involves inputs by the user make sure to provide an example of a good input.

Comment (mhamzas) [reply flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html:1]: Thanks, I'll love to write a post.
My Email address : contact@mhamzas.com

**Comment by alexed1** [comment]
Hi, Hamzas. Sorry I've taken so long on review. Note that this is now showing a minor conflict. Not a big deal, but can you resolve it?

File: flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html (hunk 4 above)
Comment (alexed1) [reply flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html:1]: I've added you as a Contributor to unofficialsf. Go ahead and draft a blog post and let me know when it's ready for review. Thanks!

**Comment by alexed1** [comment]
I'm going to go ahead and merge this. I'd still like you to write a blog post, though.

File: flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html (hunk 4 above)
Comment (mhamzas) [reply flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html:1]: Hi, 

Thanks but did not receive any email yet.

Comment (mhamzas) [reply flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html:1]: I'd also like to post the following on unofficialsf 
https://www.mhamzas.com/blog/2020/07/15/add-to-calenda/

Comment (mhamzas) [reply flow_screen_components/recordDetailFSC/force-app/main/default/lwc/recordDetailFlowScreenComponent/recordDetailFlowScreenComponent.html:1]: > Hi,
> 
> Thanks but did not receive any email yet.

Used forget pass to login

**Comment by alexed1** [comment]
Hi, Hamza. Can you take a look at these two items:
image

**Comment by mhamzas** [comment]
> Hi, Hamza. Can you take a look at these two items:
> image

Sure. Will get back to you shortly.

**Comment by mhamzas** [comment]
Added an option to allow adding classes by your own + set the default class back to : 'slds-box slds-theme_default'
@alexed1

**Pull request merged**